app/views/controllers/comments/_comment.erb

```diff
@@ -23,7 +28,7 @@ target_type = tag.span(target.class.name.to_sym.t, class: "small") \
         concat(tag.div(class: "comment-info") do
           concat(tag.span(class: "comment-author text-nowrap") do
             ["#{:BY.t}: ",
-             if defined?(controls) && controls
+             if controls
```

Comment:
The fact that we're not seeing the user links implies to me that htis is coming through as `nil` when there is a non-nil value for `@user` when the broadcast happens.